src/nix/nix.md

```diff
@@ -160,16 +164,28 @@ When the option `-f` / `--file` *path* \[*attrpath*...\] is given, installables
 If attribute paths are provided, commands will operate on the corresponding values accessible at these paths.
 The Nix expression in that file, or any selected attribute, must evaluate to a derivation.
 
+To emulate the `nix-build '<nixpkgs>' -A hello` pattern, use:
```

Comment:
It feels ad hoc / arbitrary to have this reference to the legacy CLI here, when there are numerous other migration examples that could be inserted randomly in the manual. Also, why here rather than in the `nix build` man page?

I think it would be better to have a "Migrating from the old CLI" section in the manual with lots of examples of `nix-*` invocations and their modern replacements.

Reply:
I decided to remove these examples for the time being. I think it does make sense to have them somewhere, but it's more work to collect enough of them to fill a page.